.gitignore

```diff
@@ -103,3 +103,4 @@ ENV/
 
 # mypy
 .mypy_cache/
+.idea
```

Comment:
This is Pycharm. I don't think the whole team uses it. You don't have to push this